src/account/mod.rs

```diff
@@ -89,4 +89,7 @@ pub struct Account {
     /// the sender address/es
     #[serde(rename = "incomingTransactions")]
     incoming_transactions: HashMap<TransactionId, (TransactionPayload, Vec<OutputWithMetadataResponse>)>,
+    /// Foundries for native tokens in outputs
+    #[serde(rename = "nativeTokenFoundries", default)]
+    native_token_foundries: HashMap<FoundryId, OutputWithMetadataResponse>,
```

Comment:
Do we really need the full `OutputWithMetadataResponse`?

Reply:
For this feature not, it could be useful for other things in the future, but I have nothing in my mind currently, so would also be fine to only store the immutable metadata feature